packages/datatrak-web/src/features/Survey/SurveyLayout.tsx

```diff
@@ -87,7 +87,7 @@ export const SurveyLayout = () => {
   } = useSurveyForm();
   const { handleSubmit, getValues } = useFormContext();
   const { mutate: submitSurveyResponse, isLoading: isSubmittingSurveyResponse } =
-    useSubmitSurveyResponse();
+    useSubmitSurveyResponse(from);
```

Comment:
Is this for taking you back to the tasks page after 'completing' a task?

Reply:
yes that's right